src/styles/Navbar.css

```diff
@@ -1,25 +1,44 @@
 @import url('https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap');
 
+:root {
+  --border-color: #e8e8e8;
+}
+
 .navbar {
   display: flex;
   margin: 0;
-  padding: 1.813rem 6.25rem;
+  padding: 1.813rem 8%;
   font-family: 'Montserrat', Helvetica, sans-serif;
+  border-bottom: 2px solid var(--border-color);
 }
```

Comment:
- [ ] Kindly respect padding around this navigation bar. Check here the deference :point_down: 

Your design | The template
-----|-------
![image](https://user-images.githubusercontent.com/65068771/155480002-f5fef49c-fac2-4612-8598-76b5a1d2ca55.png) | ![image](https://user-images.githubusercontent.com/65068771/155480153-2bacf54b-0134-4863-b278-af59a86caf27.png)

- [ ] Also add the icon at the right of the navigation bar because the requirements say `Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented`